app/models/manageiq/providers/inventory/persister/builder/persister_helper.rb

```diff
@@ -115,7 +116,6 @@ def shared_options
       :strategy               => strategy,
       :saver_strategy         => saver_strategy,
       :targeted               => targeted?,
-      :parent                 => parent,
```

Comment:
Moving this one up out of shared_properties because it can be changed per-inventory_collection